library/std/src/os/nto/mod.rs

```diff
@@ -0,0 +1,4 @@
+#![stable(feature = "raw_ext", since = "1.1.0")]
+
+pub mod fs;
+pub mod raw;
```

Comment:
This will hide the module to public access, but make the types defined in it available for reexport, so as to discourage reusing a deprecated module while still allowing code using `std::os::unix` to work.
```suggestion
pub(super) mod raw;
```

Reply:
Merged suggestion after testing compilation locally.